### PR TITLE
38 nodeguard withdrawals only allow to add negative btc amount

### PR DESCRIPTION
**Pull request by markettes**
Done: 
- Applied same logic to the max and min in the numeric field as for validation in save

### CODE REVIEW

File: src/Pages/Withdrawals.razor [hunk 1]
```diff
@@ -722,22 +717,22 @@
 
     private async Task RejectOrCancel()
     {
-        if ((_rejectCancelStatus == WalletWithdrawalRequestStatus.Cancelled || _rejectCancelStatus == WalletWithdrawalRequestStatus.Rejected)
```
Comment (cvicenti) [inline src/Pages/Withdrawals.razor:725]: did you perform these formatting changes on the go or  were they automatically done by your IDE? We should keep merge requests small and change only what it is described in the issue IMHO.
We can always create a new issue to address formatting issue

File: src/Pages/Withdrawals.razor [hunk 2]
```diff
@@ -440,19 +440,14 @@
...
-        {
-            ToastService.ShowError("Getting maximum of Bitcoins for withdrawal has failed");
-        }
-        
+
+        var environmentVariableMin = Environment.GetEnvironmentVariable("MINIMUM_WITHDRAWAL_BTC_AMOUNT");
+        var environmentVariableMax = Environment.GetEnvironmentVariable("MAXIMUM_WITHDRAWAL_BTC_AMOUNT");
+        if (environmentVariableMin == null) _minWithdrawal = 0.0m;
+        else _minWithdrawal = decimal.Parse(environmentVariableMin, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
+        if (environmentVariableMax == null) _maxWithdrawal = 21_000_000;
```
Comment (cvicenti) [inline src/Pages/Withdrawals.razor:448]: Ideally these magic numbers should be constants at the top of the file so we can easily find them if we need to change them

File: src/Pages/Withdrawals.razor (hunk 1 above)
Comment (markettes) [reply src/Pages/Withdrawals.razor:725]: Done by the IDE, MB I didnt payed attention to it

File: src/Pages/Withdrawals.razor (hunk 2 above)
Comment (markettes) [reply src/Pages/Withdrawals.razor:448]: I will do it

**Pull request merged**